subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/FasterIncrementalAndroidBuildsPerformanceTest.groovy

```diff
@@ -131,4 +135,24 @@ class FasterIncrementalAndroidBuildsPerformanceTest extends AbstractCrossBuildPe
 
         final String argument
     }
+
+    void applyEnterprisePlugin(GradleBuildExperimentSpec.GradleBuilder builder) {
+        builder.addBuildMutator { invocationSettings ->
+            new BuildMutator() {
+                String originalSettingsFileText
+                final File settingsFile = new File(invocationSettings.projectDir, "settings.gradle")
+
+                @Override
+                void beforeScenario(ScenarioContext context) {
+                    originalSettingsFileText = settingsFile.text
+                    GradleEnterprisePluginSettingsFixture.applyEnterprisePlugin(settingsFile)
+                }
+
+                @Override
+                void afterScenario(ScenarioContext context) {
+                    settingsFile.text = originalSettingsFileText
+                }
+            }
+        }
+    }
```

Comment:
Would it make sense to add this to `gradle-profiler`?

Reply:
Yeah, probably via an init script `gradle-profiler` can already generate.

Reply:
Are you planning to address this in this PR or shall we create an issue?

Reply:
I am not. https://github.com/gradle/gradle-profiler/issues/191